learning/features/animation/introduction_2d.rst

```diff
@@ -328,9 +328,7 @@ Godot's animation engine doesn't stop here. If you're already
 comfortable with Godot's scripting language GDScript and API you know
 that each node type is a class and has a bunch of callable functions.
 
-For example, the
-```SamplePlayer2D`` <http://docs.godotengine.org/en/stable/classes/class_sampleplayer2d.html>`__
-node type has a function to play a sample.
+For example, the ``SamplePlayer2D`` node type has a function to play a sample.
```

Comment:
It was good to make it a reference to the page, but it should not be hardcoded. Here's how to do it:
```
For example, the :ref:`class_SamplePlayer2D` node type has...
```
This will automatically be converted to a relative link to `/classes/class_sampleplayer2d.html` (without the domain and the language and branch namespaces, which we don't want to hardcode).

Reply:
The `class_SamplePlayer2D` comes from the identifier at the top of the corresponding page: https://raw.githubusercontent.com/godotengine/godot-docs/2.1/classes/class_sampleplayer2d.rst (without the leading underscore).

Reply:
> <pre>For example, the :ref:`class_SamplePlayer2D` node type has...</pre>

BTW, in the <code>:ref:`some_reference`</code>, the link's caption will be taken from the title of the page. If you need to specify a custom caption (not needed here), you can use <code>:ref:`caption <some_reference>`</code>.